CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to Gnosis Guild projects
+
+A big welcome and thank you for considering contributing to Gnosis Guild's open source projects! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
```

Comment:
issues should be plural i think. ('addressing your issues')

Reply:
Good catch.